mkt/downloads/tests/test_views.py

```diff
@@ -54,7 +56,7 @@ def test_download_already_signed(self):
         with mock.patch.object(packaged, 'sign', mock_sign):
             # Sign the app before downloading, like it would normally happen.
             self.app.sign_if_packaged(self.file.version_id)
-
+        # Don't count things left over from setup.
```

Comment:
Move that comment above, rewording it to say you are sending the tasks to make sure they don't mess with the `assertNumQueries` calls below ?